lib/query-interface.js

```diff
@@ -495,18 +495,20 @@ QueryInterface.prototype.insert = function(instance, tableName, values, options)
   });
 };
 
-QueryInterface.prototype.upsert = function(tableName, values, updateValues, model, options) {
+QueryInterface.prototype.upsert = function(tableName, valuesByField, updateValues, model, options) {
```

Comment:
Mainly for clarity. It seems to me that most of the time the values are passed around by attribute name and not by field name. My intent here was to be more informative.